Tasks/DropValidatorV0/task.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "bae07fbb-2096-488a-ab37-adf7808cc9fe",
+    "name": "DropValidator",
+    "friendlyName": "Drop validator",
+    "description": "Validates a given manifest with a manifest file to verify its authenticity.",
```

Comment:
This looks like a recurrent message in different files. Is it possible to store it in a variable?

Reply:
There are no variables in task.json files

Reply:
I see $(definition) below in the file, isn't that a kind of variable? I confess that I have never written a task and I'm not sure how this task.json is used and if there is a process to do some replace of some values or something like that

Reply:
that's specific to that particular property. It's normally used in the editor to substitute task parameters so the name of the task magically includes one or more of the parameter values. I'm not sure what "definition" would expand to though...

Reply:
fixed